man.rmd

```diff
@@ -21,7 +21,7 @@ R provides a standard way of documenting the objects in a package: you write `.R
 * It abstracts over the differences in documenting different types of objects,
   so you need to learn fewer details.
 
-As well as generating `.Rd` files, roxygen2 can also manage your `NAMESPACE` and the `Collate` field in `DESCRIPTION`. This chapter discusses `.Rd` files and the collate field. [`NAMESPACE`](#namespace) describes how you can use roxygen2 to manage your `NAMESPACE`, and why you should care.
```

Comment:
When the link is formatted as code (backticks), it's not clear from the styling that it's a hyperlink (no colour, no underlining until one hovers one's mouse over it).